packages/ui/src/app/config.ts

```diff
@@ -4,3 +4,4 @@ export const OLYMPIA_TESTNET_QUERY_NODE =
   process.env.REACT_APP_OLYMPIA_TESTNET_QUERY_NODE || 'https://olympia-dev.joystream.app/query/server/graphql'
 export const OLYMPIA_TESTNET_QUERY_NODE_SOCKET =
   process.env.REACT_APP_OLYMPIA_TESTNET_QUERY_NODE_SOCKET || 'wss://olympia-dev.joystream.app/query/server/graphql'
+export const MEMBERSHIP_FAUCET_URL = process.env.MEMBERSHIP_FAUCET_URL || 'http://localhost:4000/register'
```

Comment:
Please start the name with `REACT_APP`

In the next PR, I will add a filter to expose only variables starting with `REACT_APP`. This should help us avoid accidentally exposing a private key with the same name.

Reply:
Nice 💡